examples/cpp/counter-app/Makefile

```diff
@@ -8,7 +8,7 @@ ifndef $(abspath $(lastword $(MAKEFILE_LIST)))_MK
 $(abspath $(lastword $(MAKEFILE_LIST)))_MK := 1
 
 # uncomment to disable generate test related targets in this directory
-#TEST_TARGET := false
+TEST_TARGET := false
```

Comment:
Given that the `RECURSE_SUBMAKEFILES` bug #1574 has been resolved (and `RECURSE_SUBMAKEFILES` is set to `false` in the parent `Makefile`), it would be ideal to revert changes to this `Makefile` unless they are still required to achieve the desired effect of disabling the build.